packages/@okta/vuepress-site/docs/concepts/session/index.md

```diff
@@ -0,0 +1,53 @@
+
+### Non-privileged access use case
+
+For non-privileged access apps, such as a loyalty program catalog, sessions can be active for a long period of time (even days) and app developers don't need to implement session timeouts to restrict access. This is especially true for mobile apps where you want your users to have a seamless and frictionless user experience. In this scenario, the application session can outlive the IdP session. Application sessions can still track user interactions without the IdP session. However, when the user signs out and ends the application session, without the IdP session, the user is redirected to the IdP sign-in page and must authenticate with the IdP to gain access to the app, as before.
+
+## Terminate sessions
+
+### Local logout
+
+Local logout is the act of signing the user out of the app and terminating the application session. The user IdP session is still active and other application sessions aren't affected by terminating a specific local application session. Local logout is managed by the app. If the user wants to access the app again, they are automatically authenticated through the active IdP session into the app and the app creates a new session for the user.
+
+See [Sign users out of your app](/docs/guides/sign-users-out/-/sign-out-of-your-app/) for a guide to implement local logout with Okta.
+
+### Single logout
+
+Single logout is the act of signing the user out of the IdP and all of the apps signed in with the same IdP credentials. This results in ending the IdP session and all the associated application sessions for the user.
+
+Currently, Okta supports single logout initiated by an app, where the user signs out of an app and the Okta IdP, ending both the app and IdP sessions. However, other active application sessions for the user can still persist without the IdP session, depending on the restrictive nature of the apps. For non-privileged access apps, the user can still access the app within the scope of the application session. For example, a user can still browse through a catalog of vacation experiences and add to their cart. When this user is ready to check out and make a payment, the app can enforce a privileged access workflow and redirect the user to re-authenticate with Okta to secure an IdP session. For privileged access apps with short-lived sessions, the user is typically redirected to re-authenticate with Okta to start a new IdP session and an application session.
+
+See [Sign users out of Okta](/docs/guides/sign-users-out/-/sign-out-of-okta/) for a guide to implement Okta sign out. For Okta Admin Console app-initiated single logout configuration, see [Single Logout in applications](https://help.okta.com/en/prod/Content/Topics/Apps/Apps_Single_Logout.htm).
```

Comment:
```suggestion
See [Sign users out of Okta](/docs/guides/sign-users-out/-/sign-out-of-okta/) for more information on how to implement Okta sign-out. For Okta Admin Console app-initiated single logout configuration, see [Single Logout in applications](https://help.okta.com/en/prod/Content/Topics/Apps/Apps_Single_Logout.htm).
```

Reply:
Going to leave this paragraph as-is: "sign out" used here is not an adjective, and we didn't tell them how to implement sign out yet, so we are not providing "more information".